mitiq/lre/tests/test_lre.py

```diff
@@ -40,8 +42,27 @@ def test_lre_exp_value(degree, fold_multiplier):
     assert abs(lre_exp_val - ideal_val) <= abs(noisy_val - ideal_val)
 
 
+@pytest.mark.parametrize("circuit_type", SUPPORTED_PROGRAM_TYPES.keys())
+def test_lre_all_qprogram(circuit_type):
+    """Verify LRE works with all supported frontends."""
+    degree, fold_multiplier = 2, 3
+    circuit = benchmarks.generate_ghz_circuit(3, circuit_type)
+
+    mock_executor = Mock(side_effect=lambda _: random.random())
+
+    lre_exp_val = execute_with_lre(
+        circuit,
+        mock_executor,
+        degree=degree,
+        fold_multiplier=fold_multiplier,
+    )
+
+    assert isinstance(lre_exp_val, float)
+    assert mock_executor.call_count == 10
```

Comment:
This still pending? It's not a big deal, but it makes me feel that the test expectation was written based on what result we empirically got by running the code, instead of the other way around.

Reply:
Ah sorry, I changed this [here](https://github.com/unitaryfund/mitiq/pull/2547/files#diff-86f176b9c86340449dc49cbe0689ec2b47646820d5634eb524e3e38fc117ae31R46-R47), but not here! I'll update.